lib/reporters/cli/index.js

```diff
@@ -147,7 +148,22 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
     emitter.on('beforeRequest', function (err, o) {
         if (err || !o.request) { return; }
 
-        print('  %s %s ', colors.gray(o.request.method), colors.gray(o.request.url.toString()));
+        if (reporterOptions.showTimestamp || reporterOptions.showTimestampUtc) {
+            var currentTime = new Date();
+
+            timestamp = reporterOptions.showTimestampUtc ?
+                currentTime.toUTCString() :
+                currentTime.toLocaleString();
+            print('  %s  %s %s ',
```

Comment:
should timestamp be wrapped in `[` brackets `]`?

Reply:
Alright, I'll add brackets and see how it looks. Will commit it before the next review.